### ISSUE TITLE
Accepted Cards limited to VISA/Mastercard but other card types are still accepted

### ISSUE THREAD

**Issue by kennethkwok** [checkout/frames-android#30]
**Describe the bug**
I've limited the types of accepted cards to VISA/Mastercard, but when I enter AMEX/other types of cards the form continues to validate them as though they were accepted, and I can progress to card validation.

This is the code I've applied on the XML component
```
checkout_card_form.apply {
            mSubmitFormListener = this@CardPaymentsActivity
            setEnvironment(Environment.SANDBOX)
            setKey(appConfiguration.checkoutKey)
            setAcceptedCard(arrayOf(CardUtils.Cards.VISA, CardUtils.Cards.MASTERCARD))
            includeBilling(false)
        }
```

**To Reproduce**
1. Apply `setAcceptedCard` filter to XML component.
2. Enter a valid card number from the [testing page](https://docs.checkout.com/docs/testing) which is of a card type that is not included in `setAcceptedCard`
3. See error

**Expected behavior**
An error message should appear underneath the card number field indicating that the card type is not valid. When the user clicks on the Validate/Pay button it should refuse to progress to making a network request.

**Screenshots**
![Screenshot_20200213-115217](https://user-images.githubusercontent.com/10232494/74433253-f81d1c00-4e57-11ea-9477-2defc058b542.jpg)

**Additional context**
Version 2.1.1 of this SDK

**Comment by ioan-ghisoi-cko** [comment]
The accepted payment method option is designed as a visual hint. For now, this is not used as a hard validation. 

The idea is that in your sandbox environment, you have all schemes enabled, resulting in the Amex card being tokenised, however, in the production environment if you do not have is enabled, the tokenisation will fail, then you can handle this in your handler and display a Toast or an alert.

**Comment by kennethkwok** [comment]
@ioan-ghisoi-cko Thanks for the clarification. Will this hard feature validation be implemented in the near future? I ask because it's already available in the iOS SDK, so it would be nice to know if feature parity is high up on the priority list for the Android SDK.

**Comment by maxim-nosov-cko** [comment]
Hi @kennethkwok 

Improvement is available in Frames SDK 4.0.0+.
Unsupported card schemes will lead to validation errors.

A sample of how exactly supported card schemes should be set can be found [here]( https://github.com/checkout/frames-android/blob/master/example_app_frames/src/main/java/com/checkout/example/frames/ui/screen/MainActivity.kt#:~:text=%3D%20PaymentFormStyle()%2C-,supportedCardSchemeList%20%3D%20listOf(,),-).
